cluster/saltbase/salt/kube-proxy/initd

```diff
@@ -40,6 +40,10 @@ DAEMON_USER=root
 #
 do_start()
 {
+        # Avoid race at boot time when both monit and init.d start the service
+        PID=$(ps -ef | grep $DAEMON | grep -v grep | awk '{print $2}')
```

Comment:
I thought pidof here reports a wrong pid in this case. Just checked it, actually pidof does scan /proc and find matching name. Will update it.